engine/rsrc/mobile-device-template.plist

```diff
@@ -91,6 +91,8 @@
 		${IPAD_STATUS_BAR_HIDDEN}
 	<key>UIStatusBarStyle</key>
 		${STATUS_BAR_STYLE}
+	<key>UIStatusBarSolid</key>
```

Comment:
I'd suggest checking in the Apple plist docs about what keys applications can put in them - I suspect Apple will reserve a certain amount of namespace and so using 'UIStatusBarSolid' might not be wise. I'm sure there is a custom/app-centric plist key namespace though :)